language/de/common.php

```diff
@@ -207,6 +208,7 @@
 	'DISPLAY_MESSAGES'		=> 'Nachrichten der letzten Zeit anzeigen',
 	'DISPLAY_POSTS'			=> 'Beiträge der letzten Zeit anzeigen',
 	'DISPLAY_TOPICS'		=> 'Themen der letzten Zeit anzeigen',
+	'DOMAIN_NO_MX_RECORD_EMAIL'	=> 'Die Domain der eingegebenen E-Mail-Adresse hat keinen gültigen MX-Eintrag.',
```

Comment:
Finde ich schöner als die ursprüngliche Form

Reply:
Ob ein Otto-Normalnutzer das kapiert? Finde ich aber auch gut so.

Reply:
Das war in der alten Fassung aber auch nicht besser ;-)

Reply:
Stimmt